source/documentation/deploying_services/index.md

```diff
@@ -312,13 +314,274 @@ To restore from a snapshot:
+    Started: 2016-08-23T15:34:41Z
+    Updated: 2016-08-23T15:42:02Z
+    ```
+
+1. Run `cf env APPNAME` to see the app's environment variables and confirm that the [VCAP_SERVICES environment variable](/#system-provided-environment-variables) contains the correct service connection details.
+
+    Your app must make a [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) connection to the service. Some libraries use TLS by default, but others will need to be explicitly configured.
+
+    GOV.UK PaaS will automatically parse the ``VCAP_SERVICES`` [environment variable](/#system-provided-environment-variables) to get details of the service and then set the `DATABASE_URL` variable to the first database found.
+
+    If your app writes database connection errors to `STDOUT` or `STDERR`, you can view recent errors with ``cf logs APPNAME --recent``. See the section on [Logs](#logs) for details.
+
+### Connect to a MySQL service from your local machine
+
+We have created the [Conduit](#conduit) plugin to simplify the process of connecting your local machine to a MySQL service. To install this plugin, run the following code from the command line:
+
+`cf install-plugin conduit`
+
+Once the plugin has finished installing, run the following code in the command line to access an SQL shell for your backing service:
+
+`cf conduit SERVICE_NAME -- mysql`
+
+where `SERVICE_NAME` is a unique descriptive name for this instance of the service.
+
+Run `cf conduit --help` for more options, and refer to the [Conduit readme file](https://github.com/alphagov/paas-cf-conduit/blob/master/README.md) [external link] for more information on how to use the plugin.
+
+### Upgrade MySQL service plan
+
+You can upgrade your service plan (for example, from free to paid high availability) by running `cf update-service` in the command line:
+
+```
+cf update-service SERVICE_NAME -p NEW_PLAN_NAME
+```
+
+where `SERVICE_NAME` is a unique descriptive name for this instance of the service, and `NEW_PLAN_NAME` is the name of your new plan. For example:
+
+```
+cf update-service my-ms-service -p S-HA-dedicated-5.7
+```
+
+The plan upgrade will begin immediately and will usually be completed within about an hour. You can check the status of the change by running the `cf services` command.
+
+You can also [queue a plan upgrade](/#queue-a-plan-migration-mysql) to happen during a maintenance window to minimise service interruption.
+
+Downgrading service plans is not currently supported.
+
+### Unbind a MySQL service from your app
+
+You must unbind the MySQL service before you can delete it. To unbind the MySQL service, run the following code in the command line:
+
+`cf unbind-service APPLICATION SERVICE_NAME`
+
+where `APPLICATION` is the name of a deployed instance of your application (exactly as specified in your manifest or push command) and `SERVICE_NAME` is a unique descriptive name for this instance of the service, for example:
+
+`cf unbind-service my-app my-ms-service`
```

Comment:
I've just noticed that we appear to have mixed formatting for these kind of single-line code snippets. This, for example, renders differently from the similar thing on L543—is this expected?